src/test/regress/expected/isolation_rebalancer_deferred_drop.out

```diff
@@ -40,12 +38,11 @@ master_move_shard_placement
 
 step s2-drop-marked-shards:
     SET client_min_messages to DEBUG1;
-    CALL isolation_cleanup_orphaned_shards();
+    CALL isolation_cleanup_orphaned_resources();
 
-s1: NOTICE:  cleaned up 1 orphaned shards
 step s1-drop-marked-shards:
-    SET client_min_messages to NOTICE;
-    CALL isolation_cleanup_orphaned_shards();
+    SET client_min_messages to ERROR;
```

Comment:
Why was this needed? It seems that the NOTICE output was kind of intended in these tests.

Reply:
decision after discussion: let's add a `select * from pg_dist_cleanup where type = 'placement` call after this. Because the warning messages are not consistent anymore now that this also cleans up replication slots/subscriptions.